internal/buff/write_test.go

```diff
@@ -1,265 +0,0 @@
-// This source file is part of the EdgeDB open source project.
```

Comment:
Is this removed on purpose?

Reply:
Yes. The testing strategy going forward is to only maintain functional tests. This change broke the tests in this file so I got rid of it instead of fixing them.